src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -93,6 +98,10 @@ public class ExceptionlessSink : ILogEventSink, IDisposable {
             if (logEvent == null || !_client.Configuration.IsValid)
                 return;
 
+            var minLogLevel = _client.Configuration.Settings.GetMinLogLevel(logEvent.GetSource());
+            if (logEvent.Level.GetLevel() < minLogLevel)
+                return;
+
```

Comment:
I was just thinking randomly, will this even be called if the `restrictedToMinimumLevel` is set? Like is there any need to check this?

Reply:
Skin will not be called. Based on our previous discussion, if the server has a special log source's level setting, this still applies, so I restored this part of the code.